requirements.txt

```diff
@@ -1,2 +1,3 @@
+rpm
```

Comment:
I don't think this is necessary unless the tests are being run as part of Travis, which isn't happening now, AFAICT.

Reply:
Not necessary, but is it hurting anything?

Reply:
I'm only able to imagine two possibilities: either it is useful or it is not.

Being puzzled by its occurrence, I looked over the PR for a bit, trying to figure out how it was used and I concluded that it didn't seemed to be used at all, at least in this code base.

Assuming it is not used, it makes no sense to me for it to be merged. If it is used in some context outside this code base, it makes sense to document what that use is so that some person doesn't notice it's not used and remove it during some later infrastructure update, thereby breaking some code outside this code base that somehow relies on it.

Reply:
`tests/blackbox/testlib/utils.py`
``` python
import rpm
...
def rpm_package_version(name):
    """
    Retrieve the version of the specified package
    :param name: Name of package
    :return: String representation of version, None if package not found.
    """
    ts = rpm.TransactionSet()
    mi = ts.dbMatch('name', name)

    for i in mi:
        return i["version"].decode("utf-8")

    return None
```